From 7ed13c0e43c679dac4575f9911581665c62e6ad2 Mon Sep 17 00:00:00 2001
From: "Andrew J. Hesford" <ajh@sideband.org>
Date: Tue, 1 Feb 2022 10:18:42 -0500
Subject: [PATCH] Revert "MAINT: Raise RuntimeError if setuptools version is
 too recent."

This reverts commit eb6be7c4765665724cd12431bfefb050ba0f2d4b.
---
 setup.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/setup.py b/setup.py
index 06170df51..703fe79e1 100755
--- a/setup.py
+++ b/setup.py
@@ -80,10 +80,6 @@
 # so that it is in sys.modules
 import numpy.distutils.command.sdist
 import setuptools
-if int(setuptools.__version__.split('.')[0]) >= 60:
-    raise RuntimeError(
-        "Setuptools version is '{}', version < '60.0.0' is required. "
-        "See pyproject.toml".format(setuptools.__version__))
 
 # Initialize cmdclass from versioneer
 from numpy.distutils.core import numpy_cmdclass
-- 
2.35.0

